twisted/topfiles/8087.feature

```diff
@@ -0,0 +1 @@
+'yield from' can now be used on Deferreds, when wrapped with twisted.internet.defer.ensureDeferred.
```

Comment:
This text, while right, is a bit confusing. You don't have to wrap *every* `Deferred` with `ensureDeferred`, just the "top level" one, right?